lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1972,19 +1864,47 @@ class EngineSemanticsOwner {
         _gestureMode = GestureMode.pointerEvents;
         _notifyGestureModeListeners();
       }
-      final List<int> keys = _semanticsTree.keys.toList();
-      final int len = keys.length;
-      for (int i = 0; i < len; i++) {
-        _detachObject(keys[i]);
+      for (final EngineFlutterView view in EnginePlatformDispatcher.instance.views) {
+        view.semantics.dispose();
```

Comment:
What happens if semantics is enabled again later? Is there a way to revive `view.semantics`?

Reply:
In a production setting semantics is only enabled one-way; there's no going back. However, our tests do rely on being able to dispose of the old tree to set up a clean environment for the following tests, so the object is revivable after it is disposed. This is more like "reset" than it is "dispose". I'll rename it.